intro-HPC/gent/macros.tex

```diff
@@ -3,20 +3,22 @@
 \macro{\university}{UGent}
 \macro{\association}{AUGent}
 % HPC definitions
-\macro{\hpc}{HPC-UGent}
+\macro{\hpc}{UGent-HPC}
```

Comment:
it reads a lot better if you swap it around, doesn't really make sense now
the text is filled with

`the \hpc`

`the HPC-UGent` doesn't read correctly, `the UGent-HPC` a lot better